hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTenantConfigurationManagement.java

```diff
@@ -208,11 +213,39 @@ private static void assertMultiAssignmentsValueChange(final String key, final Jp
         }
     }
 
+    private void assertMultiAssignmentDefaultWeightValueChange(final String key,
+            final JpaTenantConfiguration valueChange) {
+        if (MULTI_ASSIGNMENTS_WEIGHT_DEFAULT.equals(key) && isMultiAssignmentEnabled() && !validateWeight(valueChange)) {
+            LOG.debug("The value {} cannot be set.", key);
+            throw new TenantConfigurationValueChangeNotAllowedException();
+        }
+    }
+
+    private boolean isMultiAssignmentEnabled() {
```

Comment:
since the method is called is...Enabled, I would not throw an exception but return false in case it it not